dist.Dockerfile

```diff
@@ -2,12 +2,16 @@ FROM python:3.9-slim
 
 # the UID and GID to run cartography as
 # (https://github.com/hexops/dockerfile#do-not-use-a-uid-below-10000).
-ARG uid=10001
-ARG gid=10001
+ARG uid=0 #10001
+ARG gid=0 #10001
```

Comment:
Why are you switching this to 0? We're purposely setting the uid to something more secure.